angular-chart.js

```diff
@@ -193,13 +202,16 @@
               getData(scope.labels, scope.data, scope.colours);
             var options = angular.extend({}, ChartJs.getOptions(type), scope.options);
             chart = new ChartJs.Chart(ctx)[type](data, options);
-            scope.$emit('create', chart);
+            scope.$emit('chart-created', chart);
```

Comment:
Will do so. Actually wanted to do that, and created a pull request after the first commit, but the pull request included the other change as well. Will take a little while